### ISSUE TITLE
feature request :plz add GPU support for float16 in tf.nn.lrn

### ISSUE THREAD

**Issue by B3N50N** [tensorflow/tensorflow#18834]
Have I written custom code yes
OS Platform and Distribution ubuntu16.04
TensorFlow installed from source
TensorFlow version:1.7
Bazel version 0.11.1
CUDA/cuDNN version 9
GPU model and memory TITAN Xp
Exact command to reproduce:N/A

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 31 days with no activity and the `awaiting response` label was assigned. Is this still an issue?
